ast2vec/repo2/base.py

```diff
@@ -77,21 +72,19 @@ def clone_repository(self, url, save_dir=None):
             target_dir = tempfile.mkdtemp(prefix="repo2base-", dir=self._tempdir)
         else:
             target_dir = save_dir
-        git_log_file = tempfile.mkstemp(prefix="gitlog-", dir=self._tempdir)[1]
-
         url = type(self).prepare_reponame(url)
         env = os.environ.copy()
         env["GIT_TERMINAL_PROMPT"] = "0"
         self._log.info("Cloning from %s...", url)
         try:
-            subprocess.check_call(
+            subprocess.check_output(
                 ["git", "clone", "--depth=1", url, target_dir],
-                env=env, stdin=subprocess.DEVNULL, stderr=open(git_log_file, 'w'))
+                env=env, stdin=subprocess.DEVNULL, stderr=subprocess.STDOUT)
         except Exception as e:
```

Comment:
I think we should catch subprocess.CalledProcessError here, otherwise e.output does not exist